README.md

```diff
@@ -0,0 +1,49 @@
+# Source for [rust-for-rustaceans.com](https://rust-for-rustaceans.com)
+
+Home of Reviews and Errata for ["Rust for Rustaceans" by Jon Gjengset](https://nostarch.com/rust-rustaceans).
+
+![book cover art with excited Ferris (Rust mascot)](RustforRustaceans_cover.png?raw=true "Rust for Rustaceans")
+
+# Submit Errata
+
+Found a bug?
+
+Just take a look at [accepted Errata pull requests](https://github.com/jonhoo/rust-for-rustaceans.com/pulls?q=is%3Apr+is%3Aclosed+errata) and submit in similar form.
+
+## Example
+
+Branch / Commit / PR name: "errata print01-ch{nn}-lst{nn} syntax error in example"
+
+````markdown
```

Comment:
Note: extra backticks `` ```` `` because the body also contains backticks.

Reply:
Ah, TIL that markdown lets you increase the number of fences in this way!